resources/js/Components/WikidataToolFooter.vue

```diff
@@ -72,6 +72,8 @@ export default Vue.extend({
 
 .footer-container {
     background-color: $color-base-90;
+    flex-grow: 0;
+    flex-shrink: 0;
```

Comment:
I think these rules are actually not needed in our case, since the footer sits outside of main in the html of the `Layout.vue` component. When I disable them in the devtools, it has no apparent affect.